### PR TITLE
Fix lint for Python 3 and 2

### LINKED ISSUE #30: Fix Lint and tests for Python 3 and 2

### PR DESCRIPTION
**Pull request by hericlesme**
## Description

- remove duplicate import (ml engine)
- pylint disable broad-except (big query) 
- fix some pylint warnings (cloud functions)

Fixes #30 

## Type of change

Please mark options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- [x] `make install`
- [x] `make ci`

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -24,6 +24,6 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 
 ## Checklist:
 
-- [ ] I did follow the [contribution guidelines](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md)
+- [ ] I did follow the [contribution guidelines](https://github.com/globocom/gcloud-utils/blob/master/CONTRIBUTING.md)
```
Comment (dmvieira) [inline pull_request_template.md]: thx

File: gcloud_utils/bigquery/bigquery.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#pylint: disable=too-many-arguments,bare-except
+#pylint: disable=too-many-arguments,bare-except, broad-except
```
Comment (dmvieira) [inline gcloud_utils/bigquery/bigquery.py]: why do you remove this lint?

Comment (hericlesme) [reply gcloud_utils/bigquery/bigquery.py]:
```
def create_table(self, dataset_id, table_id):
    """Create a table based on dataset"""
    try:
        self.create_dataset(dataset_id)
    except Exception:
        pass
```

@dmvieira It's a temporary removal until I know what the real type of exception is going to be caught here. Pylint accuses the use of `except: Exception`. But if you know what type I can fix.

Comment (dmvieira) [reply gcloud_utils/bigquery/bigquery.py]: @alexandreyy do you know about this Exception?

Comment (alexandreyy) [reply gcloud_utils/bigquery/bigquery.py]: @dmvieira No. We should remove that code or handle that Exception.

Comment (hericlesme) [reply gcloud_utils/bigquery/bigquery.py]: So, what should I do?

Comment (alexandreyy) [reply gcloud_utils/bigquery/bigquery.py]: I prefer to remove it. @dmvieira What do you think?

Comment (hericlesme) [reply gcloud_utils/bigquery/bigquery.py]: @alexandreyy remove the create_table function?

Comment (alexandreyy) [reply gcloud_utils/bigquery/bigquery.py]: @hericlesme The Exception

Comment (hericlesme) [reply gcloud_utils/bigquery/bigquery.py]: @alexandreyy Okay. Can I do this? Or should I wait for @dmvieira?

Comment (alexandreyy) [reply gcloud_utils/bigquery/bigquery.py]: @dmvieira @hericlesme wait :)

**Pull request closed without merging**